docs/process/ci_build.md

```diff
@@ -28,15 +28,24 @@ Basic concepts:
 
-Less frequently, there might be specific changes that cannot be tested via a PR because they only happen on merges to the main branch `master`. For example, API docs changes and benchmark dashboard changes should only occur on merges to master. In this case, you can use your personal fork of the upstream repo as a testing ground. The naive approach is not recommended -- to directly modify your `master` branch -- because it requires awkwardly changing your git repo during and after testing in ways that are often confusing and opposite to "git-flow" habits. An alternative strategy for using your fork to do testing is, instead of making feature changes directly on `master`, you can maintain the changes on a feature branch as normal for a "git-flow" workflow. And you can create a special testing branch from the feature branch that maintains an extra commit at its head that loosens restrictions and makes fork-specific configuration changes. For example, when testing fixes to dashboards in `unicode-org/icu4x`, a feature branch `ci-bench-per-component` was created. Then, a testing branch [`ci-bench-per-component-test`](https://github.com/echeran/icu4x/pull/13/commits) was created. Feature changes were made on the feature branch, and the `*-test` branch was always rebased on top of the feature branch in order to maintain fork/testing reconfigurations, such as [allowing `master`-only jobs to run on all branches](https://github.com/echeran/icu4x/pull/13/commits/5e83cb4f97a6629b9bfa330aaeb73b3c523f0a8f), making the [API docs / benchmarks copy destination to be on the personal fork](https://github.com/echeran/icu4x/pull/13/commits/61953d9275d3853d96f43ed95e2ff0e9dc5fdd5f), and adding lots of stdout debugging that gets captured in the logs ([example](https://github.com/echeran/icu4x/pull/13/commits/d6168715ac644b785c920b639cd5238e7a52f806)). The `*-test` branch (`ci-bench-per-component-test`) was pushed to the personal fork after each change to trigger workflow test invocations. The feature branch was largely kept locally until the feature work was done, but the important point was to preserve the invariant that the feature branch (`ci-bench-per-component`) was always in a good state to be turned into a PR on the upstream repo.
+There might be reasons why you want or need to test changes to GH Actions on your personal fork. Less frequently, there might be specific changes that cannot be tested via a PR because they only happen on merges to the main branch `master`. For example, API docs changes and benchmark dashboard changes should only occur on merges to master. In this case, you can use your personal fork of the upstream repo as a testing ground. The naive approach is not recommended -- to directly modify your `master` branch -- because it requires awkwardly changing your git repo during and after testing in ways that are often confusing and opposite to "git-flow" habits. 
```

Comment:
Suggestion: I think abbreviations here make the wording harder to interpret as a new reader. I would prefer to use the full name of "GitHub Actions" everywhere.

Reply:
Done.